setup.py

```diff
@@ -23,14 +24,19 @@ def find_version(*file_paths):
         return version_match.group(1)
     raise RuntimeError("Unable to find version string.")
 
+def pillow_requirement():
+    if platform.system() == 'Darwin':
+        return 'pillow >= 4.1.1, != 5.1'
+    else: 
```

Comment:
Seems you have white-space at the end. You need to remove it to pass the Lint check as well.